iam-login-service/src/main/java/it/infn/mw/iam/api/tokens/Constants.java

```diff
@@ -0,0 +1,11 @@
+package it.infn.mw.iam.api.tokens;
+
+public class Constants {
+
+  public static final String ACCESS_TOKENS_ENDPOINT = "/iam/api/access-tokens";
+  public static final String REFRESH_TOKENS_ENDPOINT = "/iam/api/refresh-tokens";
+
+  private Constants() {
+    throw new IllegalStateException("Utility class");
```

Comment:
I would leave the private constructor empty, or containing a comment like: utility class, it should not be instantiated.